sycl/include/sycl/vector.hpp

```diff
@@ -560,7 +543,10 @@ template <typename T> struct LShift {
 // Actual computation performed on conversion or assignment operators.
 template <typename VecT, typename OperationLeftT, typename OperationRightT,
           template <typename> class OperationCurrentT, int... Indexes>
-class SwizzleOp {
+class SwizzleOp : public detail::NamedSwizzlesMixinBoth<
```

Comment:
I'm using "Both" vs "Const" here because that's what existing implementation does. Per the specification it should be `const` only version - to be cleaned up separately under `-fpreview-breaking-changes`.